src/ir/context.rs

```diff
@@ -160,6 +161,10 @@ pub struct BindgenContext<'ctx> {
     /// uses. See `ir::named` for more details. Always `Some` during the codegen
     /// phase.
     used_template_parameters: Option<HashMap<ItemId, ItemSet>>,
+
+    /// The set of `TypeKind::Comp` items found during parsing that need their
+    /// bitfield allocation units computed. Drained in `compute_bitfield_units`.
+    need_bitfield_allocation: Vec<ItemId>,
```

Comment:
We need to delay the bitfield allocation unit computation until after we have resolved type refs because a bitfield could be of some user defined `enum` type that we haven't resolved yet.

We could potentially further slim down this work list by eagerly going `RawField` -> `Field::DataMember` for `CompInfo`s that don't have *any* bitfields, but I don't think it will have a large impact on perfomance.